source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,22 @@
+RoboRIO2 Imaging
+================
+
+The roboRIO2 uses a microSD card to hold all of it's software and robot code.
+
+Getting the image
+-----------------
+
+The image is provided as part of the FRC Game Tools install.  To locate the image you can open the Imaging Tool
+
+Copy the image to your MicroSD card
+-----------------------------------
+
+Download and install `Etcher <https://www.balena.io/etcher/>`__ to image the micro SD card. The micro SD card needs to be at least
+4 GB. Note: a `micro SD to USB dongle <https://www.amazon.com/gp/product/B0779V61XB>`__ works well for writing to micro SD cards.
```

Comment:
I know the default card that comes with the roboRIO2 is 4gb, but I'm guessing smaller cards could be used. The roboRIO 1 only had 512 mb of storage, and the zipped image is only 130mb.